File: module_home_tab_task/src/main/java/com/hengwei/module_home_tab_task/utils/AppUtils.java
package com.hengwei.module_home_tab_task.utils;

import android.util.Log;

import com.geen.commonlibary.utils.ToastUtil;

public class  AppUtils {

    public static void showLog(String log){
        Log.e("xxx",log);
        ToastUtil.showTips(log);
    }
}

File: module_home_tab_task/src/main/java/com/hengwei/module_home_tab_task/utils/AppUtils.kt
package com.hengwei.module_home_tab_task.utils

import android.util.Log
import com.geen.commonlibary.utils.ToastUtil

object AppUtils {
    @JvmStatic
    fun showLog(log: String?) {
        Log.e("xxx", log)
        ToastUtil.showTips(log)
    }
}

fun test(){

}